code/modules/vore/eating/belly_vr.dm

```diff
@@ -465,10 +465,16 @@
 			owner << "<span class='warning'>You feel your [name] start to cling onto its contents...</span>"
 			digest_mode = DM_ABSORB
 			return
-
-		else if(prob(digestchance) && digest_mode != DM_DIGEST) //Finally, let's see if it should run the digest chance.
+		
+		else if(prob(digestchance) && digest_mode != (DM_ITEMWEAK | DM_DIGEST)) //Finally, let's see if it should run the digest chance.
```

Comment:
digest_mode is not a bitfield.  You cannot use bitwise OR to test it.    I think you mean   `else if(prob(digestchance) && !(digest_mode == DM_ITEMWEAK || digest_mode == DM_DIGEST))`